packages/internals/src/get-generators/getGenerators.ts

```diff
@@ -495,3 +497,18 @@ In case you want to fix this, you can provide ${chalk.greenBright(
     }
   }
 }
+
+function filterGenerators(generators: GeneratorConfig[], generatorNames: string[]) {
+  if (generatorNames.length < 1) {
+    return generators
+  }
+
+  const filtered = generators.filter((generator) => generatorNames.includes(generator.name))
+
+  if (filtered.length !== generatorNames.length) {
+    const missing = generatorNames.find((name) => filtered.find((generator) => generator.name === name) == null)
+    throw new Error(`Generator ${chalk.bold(missing)} is not found.`)
```

Comment:
Can missing be more than 1 generator here actually?
Maybe we could have something like
"The specified generator(s) `${chalk.bold(generatorNames.join(', ')}` do not exist in your Prisma schema."

Reply:
@Jolg42 Sounds good! 386ef65